_posts/2017-06-27-ardublocklysenseBox.md

```diff
@@ -0,0 +1,58 @@
+
+Unter [Arbublockly](https://sensebox.de/blockly) befindet sich eine online Version der Oberfläche die ohne Installation auf den gängigen Browsern läuft. Hierbei muss allerdings der generierte Code mittels Copy and Paste in die Arduino IDE kopiert werden.
+
+Die offline Varianten der Ardublockly Oberfläche befinden sich im [Github Repository](github.com/senseBox/ardublockly) in den [Releases](https://github.com/sensebox/ardublockly/releases)  (die Releases werden in unregelmäßigen Abständen neu veröffentlicht). Hier gibt es für die gängigen Betriebssysteme (Windows, MacOS, Linux) eine jeweilige Version.
+Für die Offline Variante muss Python (mind. Version 2.7) installiert sein. Weitere Informationen und der Download der Software befinden sich auf der [Python Homepgage](https://www.python.org/). Unter MacOS ist Python bereits vorinstalliert. Hierbei muss nur die Ardublock.app in den Programmordner kopiert werden.
+Damit der direkte Upload oder das direkte öffnen des Sketches in der Arduino IDE funktioniert, muss der Programmpfad in den Einstellungen gesetzt werden.
+
+Damit die Sensoren ausgelesen werden können wird eine Software Bibliothek benötigt, diese beeinhaltet die Software für alle Sensoren, die in der senseBox:edu enthalten sind ([senseBox-Library](https://github.com/sensebox/senseBox_library)). Die Software Library muss heruntergeladen werden und in die Arduino IDE integriert werden (mehr Informationen dazu [hier](https://edu.books.sensebox.de/de/getting_started/installation_der_software.html)).
+
+Verwendung der Oberfläche
+============
+Der Screenshot zeigt den Grundaufbau der Ardublockly Oberläche. Im Auswahlmenü rechts können die Programmierblöcke herausgezogen und in der Programmieroberfläche zusammengesetzt werden. Es empfiehlt sich mit dem Block "Arduino Führe zuerst aus/Arduino Endlosschleife" aus dem Bereich Funktionen zu starten.
+
+Im rechten Fenster wird der generierte Code angezeigt. Mithilfe dieser Ansicht lässt sich nachvollziehen welche Programmierbefehle hinter den Blöcken stehen. Neben den senseBox Blöcken gibt es eine Vielzahl von verschiedenen Blöcken die es ermöglichen komplexe Proramme zu schreiben ohne die Syntax der Programmiersprache mühsam erlernen zu müssen. Mithilfe der Ardublockly Oberfläche können auch schon jüngere Nachwuchswissenschaftler mit der senseBox:edu arbeiten. Durch das Parallele Anzeigen des Programmcodes wird der Schritt zu den ersten Textbasierten Programmen deutlich erleichtert.
+{% include image.html image=page.image2 %}
+
+Ausblick
+============
+Wir arbeiten zurzeit an einem Buch (ähnlich unserer Anleitung zur [senseBox:edu](https://edu.books.sensebox.de/de/)) speziell für die Ardublockly Oberfläche. Durch die starke Vereinfachung können nun die ersten Projekte in kürzerer Zeit durchgeführt werden und es bleibt mehr Zeit im Unterricht oder in Projekttagen um die Messgeräte zu verwenden oder das Projekt noch komplexer auszubauen.
```

Comment:
... mehr Zeit im Unterricht oder in Projekttagen, um die Messgeräte zu verwenden ...